msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -121,16 +154,39 @@ The RSC permissions are declared in your app manifest (JSON) file.  Add a [webAp
   }
 ```
 
+### Example for RSC in a chat
+```json
+"webApplicationInfo": {
+    "id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
+    "resource": "https://RscBasedStoreApp",
+    "applicationPermissions": [
+      "ChatSettings.Read.Chat",
+      "ChatSettings.ReadWrite.Chat",
+      "ChatMessage.Read.Chat",
+      "ChatMember.Read.Chat",
+      "Chat.Manage.Chat",
+      "TeamsTab.Read.Chat",
+      "TeamsTab.Create.Chat",
+      "TeamsTab.Delete.Chat",
+      "TeamsTab.ReadWrite.Chat",
+      "TeamsAppInstallation.Read.Chat",
+      "OnlineMeeting.ReadBasic.Chat"
+    ]
+  }
+```
+
 ## Sideload your app in Teams
```

Comment:
Note: If the same app is meant to support installation in both team and chat scopes, then both RSC permissions for both chats and teams can be specified in the the same 'applicationPermissions' section of the manifest.

Please wordsmith as necessary.